### DISCUSSION TITLE
Pevious MEDS ETL not running

### DISCUSSION THREAD

**Discussion by bnevesmd** [som-shahlab/femr#225] [Q&A]
HI there. We are trying to convert our OMOP data to MEDS format  in order to use CLMBR-T. We realize older version of MEDS 0.1.1  is required, however we are keeping having the following error when running meds_etl:

`Generating metadata from OMOP `concept` table
100%|█████████████████████████████████████████████| 1/1 [00:09<00:00,  9.29s/it]
Finding birth and death codes:  92%|▉| 8340025/9093629 [00:02<00:00, 3410987.07i
Decompressing OMOP tables, mapping to MEDS Flat format, writing to disk...
  0%|                                                     | 0/6 [00:00<?, ?it/s]/anaconda/envs/embeddings/lib/python3.9/site-packages/meds_etl/omop.py:169: PerformanceWarning: Determining the column names of a LazyFrame requires resolving its schema, which is a potentially expensive operation. Use `LazyFrame.collect_schema().names()` to get the column names without this warning.
  pl.col(table_name + option) for option in options if table_name + option in batch.columns
[/anaconda/envs/embeddings/lib/python3.9/site-packages/meds_etl/omop.py:196]: PerformanceWarning: Determining the column names of a LazyFrame requires resolving its schema, which is a potentially expensive operation. Use `LazyFrame.collect_schema().names()` to get the column names without this warning.
  if concept_id_field.replace("_concept_id", "_source_concept_id") in batch.columns:
  0%|                                                     | 0/6 [00:03<?, ?it/s]
Traceback (most recent call last):
  File "/anaconda/envs/embeddings/bin/meds_etl_omop", line 8, in <module>
    sys.exit(main())
  File "/anaconda/envs/embeddings/lib/python3.9/site-packages/meds_etl/omop.py", line 704, in main
    process_table(task)
  File "/anaconda/envs/embeddings/lib/python3.9/site-packages/meds_etl/omop.py", line 216, in process_table
    code = concept_id.map_dict(concept_id_map)
AttributeError: 'Expr' object has no attribute 'map_dict'`

Thank you!

**Comment by EthanSteinberg** [comment]
Hi bnevesmd,

Can you try meds_etl == 0.1.3 instead of 0.1.1?

I think that should fix the issue you were facing. I'll get the documentation on our side updated to point to the right versions.
